[pytorch] update docker image version

ljk53 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #30937 [pytorch] integrate op dependency analysis process into CI
* **#31848 [pytorch] update docker image version**
* #32027 [pytorch] skip TEST_DILL on Python2
* #32011 [pytorch] gate torch_global_deps with BUILD_SHARED_LIBS flag

Summary:
Trigger docker image build and bump up docker image version.

Test Plan:
- Check tag at: http://ossci-docker.s3-website.us-east-1.amazonaws.com/pytorch.html

Differential Revision: [D19282725](https://our.internmc.facebook.com/intern/diff/D19282725)

ljk53 committed:
[pytorch] update docker image version

Summary:
Trigger docker image build and bump up docker image version.

Test Plan:
- Check tag at: http://ossci-docker.s3-website.us-east-1.amazonaws.com/pytorch.html

[ghstack-poisoned]

kostmo commented:
## :pill: CircleCI build failures summary and remediations
As of commit 10b14cd3:


* **7/8** failures introduced in this PR
* **1/8** recognized as flaky :snowflake:
    * Re-run these jobs?


## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=10b14cd348d089d4e74d07495e3eb1cb991b11a3).

### :snowflake: 1 failure recognized as flaky
The following build failures have been detected as flaky and **may not be your fault**:
#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/4124008) pytorch-linux-xenial-py3.6-clang7 (1/1)
**Step:** "build_docker_image_pytorch-linux-xenial-py3.6-clang7" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=80034709) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=80034709)) :snowflake:
<details>
<summary>
<code>$<3>$<3> 79554e8b009a: Waiting  $<3>$<3> 395e4aa98b72: Waiting  toomanyrequests: Rate exceeded</code>
</summary>

```
$<3>$<3> 42f26ed810f6: Preparing   
$<3>$<3> 1c11a99155ed: Preparing   
$<3>$<3> 2d6a13684848: Preparing   
$<3>$<3> 62bca4030311: Preparing   
$<3>$<3> 9d19bc7ce3d6: Preparing   
$<3>$<3> 490d801da324: Preparing   
$<3>$<3> b5adf699711c: Preparing   
$<3>$<3> 77008e118980: Preparing   
$<3>$<3> 6cb741cb00b7: Preparing   
$<3>$<3> f36b28e4310d: Preparing   
79554e8b009a: Waiting  $<3>$<3> 395e4aa98b72: Waiting  toomanyrequests: Rate exceeded 
+ docker logout 308535385114.dkr.ecr.us-east-1.amazonaws.com 
Removing login credentials for 308535385114.dkr.ecr.us-east-1.amazonaws.com 
```

</details>



---

<details><summary>This comment was automatically generated by <a href="https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment">Dr. CI</a> (expand for details).</summary>Follow <a href="https://dr.pytorch.org/admin/comments-opt-out.html">this link to opt-out</a> of these comments for your Pull Requests.

Please report bugs/suggestions on the <a href="https://github.com/kostmo/circleci-failure-tracker/issues">GitHub issue tracker</a>.
</details>

ljk53 committed:
Update on "[pytorch] update docker image version"


Summary:
Trigger docker image build and bump up docker image version.

Test Plan:
- Check tag at: http://ossci-docker.s3-website.us-east-1.amazonaws.com/pytorch.html

[ghstack-poisoned]

ljk53 commented:
@pytorch/buildbot rerun tests

ljk53 committed:
Update on "[pytorch] update docker image version"


Summary:
Trigger docker image build and bump up docker image version.

Test Plan:
- Check tag at: http://ossci-docker.s3-website.us-east-1.amazonaws.com/pytorch.html

[ghstack-poisoned]

ljk53 committed:
Update on "[pytorch] update docker image version"


Summary:
Trigger docker image build and bump up docker image version.

Test Plan:
- Check tag at: http://ossci-docker.s3-website.us-east-1.amazonaws.com/pytorch.html

[ghstack-poisoned]

ljk53 committed:
Update on "[pytorch] update docker image version"


Summary:
Trigger docker image build and bump up docker image version.

Test Plan:
- Check tag at: http://ossci-docker.s3-website.us-east-1.amazonaws.com/pytorch.html

[ghstack-poisoned]

ljk53 committed:
Update on "[pytorch] update docker image version"


Summary:
Trigger docker image build and bump up docker image version.

Test Plan:
- Check tag at: http://ossci-docker.s3-website.us-east-1.amazonaws.com/pytorch.html

Differential Revision: [D19282725](https://our.internmc.facebook.com/intern/diff/D19282725)

[ghstack-poisoned]

ljk53 committed:
Update on "[pytorch] update docker image version"


Summary:
Trigger docker image build and bump up docker image version.

Test Plan:
- Check tag at: http://ossci-docker.s3-website.us-east-1.amazonaws.com/pytorch.html

Differential Revision: [D19282725](https://our.internmc.facebook.com/intern/diff/D19282725)

[ghstack-poisoned]

ezyang commented on .circleci/cimodel/data/pytorch_build_definitions.py:
```diff
@@ -13,7 +13,7 @@
 # ARE YOU EDITING THIS NUMBER?  MAKE SURE YOU READ THE GUIDANCE AT THE
 # TOP OF .circleci/config.yml
-DOCKER_IMAGE_VERSION = "405"
+DOCKER_IMAGE_VERSION = "a8006f9a-272d-4478-b137-d121c6f05c83"
```
oh, what a brave new world we live in

ljk53 committed:
Update on "[pytorch] update docker image version"


Summary:
Trigger docker image build and bump up docker image version.

Test Plan:
- Check tag at: http://ossci-docker.s3-website.us-east-1.amazonaws.com/pytorch.html

Differential Revision: [D19282725](https://our.internmc.facebook.com/intern/diff/D19282725)

[ghstack-poisoned]

ljk53 committed:
Update on "[pytorch] update docker image version"


Summary:
Trigger docker image build and bump up docker image version.

Test Plan:
- Check tag at: http://ossci-docker.s3-website.us-east-1.amazonaws.com/pytorch.html

Differential Revision: [D19282725](https://our.internmc.facebook.com/intern/diff/D19282725)

[ghstack-poisoned]

ljk53 committed:
Update on "[pytorch] update docker image version"


Summary:
Trigger docker image build and bump up docker image version.

Test Plan:
- Check tag at: http://ossci-docker.s3-website.us-east-1.amazonaws.com/pytorch.html

Differential Revision: [D19282725](https://our.internmc.facebook.com/intern/diff/D19282725)

[ghstack-poisoned]
